src/AssetsPage/index.jsx

```diff
@@ -17,11 +20,31 @@ class AssetsPage extends React.Component {
       });
   }
 
-  render() {
+  constructor(props) {
+    super(props);
+    this.state = {
+      checked: 'off',
+    };
     AssetsPage.request();
+  }
+
+  handleCheckBoxChange = (event) => {
```

Comment:
I like this style versus having to do `this.handleCheckboxChange = this.handleCheckboxChange.bind(this)` in the constructor. It requires the "transform-class-properties" babel plugin which I added. If you all don't agree let me know.